alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md

```diff
@@ -0,0 +1,24 @@
+# AlertmanagerClusterFailedToSendAlerts
+
+## Meaning
+
+The alert `AlertmanagerClusterFailedToSendAlerts` is triggered when **all**
+Alertmanager instances in a cluster has consistently failed to send
+notifications to a critical integration.
+
+## Impact
+
+Alerts to the integration are not delivered.
+
+## Diagnosis
+
+Check the logs of the `alertmanager-main` pods in the `openshift-monitoring`
+namespace:
+
+```console
+$ oc -n openshift-monitoring logs -l 'alertmanager=main'
+```
+
+## Mitigation
+
+The resolution depends on the particular issue reported in the logs.
```

Comment:
Maybe list a few reasons why the alert might exist like unreachable endpoint, misconfigured address or credentials, ...?